content/en/docs/concepts/windows/user-guide.md

```diff
@@ -186,15 +186,15 @@ so taints and tolerations (or node selectors) are still required
 ### Ensuring OS-specific workloads land on the appropriate container host
 
 Users can ensure Windows containers can be scheduled on the appropriate host using taints and tolerations.
-All Kubernetes nodes running Kubernetes {{< skew currentVersion >}} have the following default labels:
+All Kubernetes nodes running Kubernetes {{< skew currentPatchVersion >}} have the following default labels:
 
 * kubernetes.io/os = [windows|linux]
 * kubernetes.io/arch = [amd64|arm64|...]
 
 If a Pod specification does not specify a `nodeSelector` such as `"kubernetes.io/os": windows`,
 it is possible the Pod can be scheduled on any host, Windows or Linux.
 This can be problematic since a Windows container can only run on Windows and a Linux container can only run on Linux.
-The best practice for Kubernetes {{< skew currentVersion >}} is to use a `nodeSelector`.
+The best practice for Kubernetes {{< skew currentPatchVersion >}} is to use a `nodeSelector`.
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.